README.md

```diff
@@ -749,7 +750,8 @@ SitePrism allows adding sections to sections) is to call the `section`
 method. It takes 3 arguments: the first is the name of the section as
 referred to on the page (sections that appear on multiple pages can be
 named differently). The second argument is the class of which an
-instance will be created to represent the page section, and the third
+instance will be created to represent the page section (this argument is optional
+and by default SitePrism will try to use `#{name}Section` format), and the third
```

Comment:
This sentence is now about 4 lines long and contains two different parenthesized sub-sentences.  Maybe it could be broken up somewhat so it's a little more readable?